app/models/application_record.rb

```diff
@@ -1,6 +1,7 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
 
+  STATE_INTAKE_CLASS_NAMES = %w[StateFileAzIntake StateFileNyIntake].map(&:to_s).freeze
```

Comment:
Moving this here to make it available to all our models instead of just `EfileSubmission`